stephan-livera-podcast/2023-01-14-james-o’beirne.md

```diff
@@ -0,0 +1,291 @@
+---
+title: A New Way to HODL?
+transcript_by: Stephan Livera
+categories: ['podcast']
+speakers: ['James O’Beirne']
```

Comment:
curly apostrophes are no good
```suggestion
speakers: ['James O'Beirne']
```

Please change the file name and remove the apostrophe all together